configs/aro/site_config.py

```diff
@@ -104,6 +104,7 @@
     'periodic_focus_time' : 0.5, # This is a time, in hours, over which to bypass automated focussing (e.g. at the start of a project it will not refocus if a new project starts X hours after the last focus)
     'stdev_fwhm' : 0.5, # This is the expected variation in FWHM at a given telescope/camera/site combination. This is used to check if a fwhm is within normal range or the focus has shifted
     'focus_exposure_time': 15, # Exposure time in seconds for exposure image
+    'focus_trigger' : 0.5, # What FWHM increase is needed to trigger an autofocus
```

Comment:
No, it will vary by site. For good sites, the atmosphere is more stable. Watery ground based sites the focus can be more variable/acquatic. Some optical tubes will have more flop and some focuser mechanisms slip.... so different sites will have different values. 0.5 is a good starting point though.

Reply:
Okay sounds good, thanks for clarifying!